pommerman/envs/v0.py

```diff
@@ -171,10 +178,17 @@ def step(self, actions):
         self._step_count += 1
         return obs, reward, done, info
 
-    def render(self, mode='human', close=False, record_pngs_dir=None, record_json_dir=None):
+    def render(self, mode=None, close=False, record_pngs_dir=None, record_json_dir=None):
         if close:
             self.close()
             return
+        
+        mode = mode
+
+        if self._mode is not None:
+            mode = self._mode
+        elif mode is None:
+            mode = 'human'
```

Comment:
if the user specifies the render, we want that to override, so
```mode = mode or self._mode```
works if you set ```self._mode = 'human'``` above.